src/pages/account/settings/components/security.tsx

```diff
@@ -0,0 +1,105 @@
+import { FormattedMessage, formatMessage } from 'umi-plugin-react/locale';
+import React, { Component, Fragment } from 'react';
+
+import { List } from 'antd';
+
+type Unpacked<T> = T extends (infer U)[] ? U : T;
+
+const passwordStrength = {
+  strong: (
+    <span className="strong">
+      <FormattedMessage id="accountandsettings.security.strong" defaultMessage="Strong" />
+    </span>
+  ),
+  medium: (
+    <span className="medium">
+      <FormattedMessage id="accountandsettings.security.medium" defaultMessage="Medium" />
+    </span>
+  ),
+  weak: (
+    <span className="weak">
+      <FormattedMessage id="accountandsettings.security.weak" defaultMessage="Weak" />
+      Weak
+    </span>
+  ),
+};
+
+class SecurityView extends Component {
+  getData = () => [
+    {
+      title: formatMessage({ id: 'accountandsettings.security.password' }, {}),
+      description: (
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'description' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)